nbproject/platform.properties

```diff
@@ -1,15 +1,12 @@
 branding.token=tmcbeans
 cluster.path=\
-    ${nbplatform.active.dir}/cnd:\
     ${nbplatform.active.dir}/extide:\
-    ${nbplatform.active.dir}/dlight:\
     ${nbplatform.active.dir}/harness:\
     ${nbplatform.active.dir}/ide:\
     ${nbplatform.active.dir}/java:\
     ${nbplatform.active.dir}/javafx:\
     ${nbplatform.active.dir}/nb:\
     ${nbplatform.active.dir}/platform:\
-    ${nbplatform.active.dir}/webcommon:\
```

Comment:
This is also something Netbeans does sometimes automatically and it should not be in commits.